pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/unnecessary/xml/UselessParentheses.xml

```diff
@@ -370,4 +370,81 @@ public class Useless {
 }
         ]]></code>
     </test-code>
+
+    <test-code>
```

Comment:
No it's not... this is still the same test-case as https://github.com/oowekyala/pmd/blob/2d5f1355c2fde0ed3eb12c99def8168ec4294c14/pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/unnecessary/xml/UselessParentheses.xml#L100-L111